tests/wicked/before_test.pm

```diff
@@ -41,6 +41,8 @@ sub run {
         systemctl('restart network');
     }
     $self->get_from_data('wicked/ifbind.sh', '/bin/ifbind.sh', executable => 1);
+    assert_script_run("ifbind.sh unbind eth0");
```

Comment:
maybe better just move it inside t01_standalone_card_ifdown_ifreload.pm ?

Reply:
I was thinking this "magic" is actually needed by all the tests, therefore I put it here...